journal/original/14-chapter-11.adoc

```diff
@@ -106,6 +111,9 @@ how contrary they acted therein to the apostles`' doctrine and practice,
 and to bring them to more moderation.
 Thus it was:--
 
+[.embedded-content-document.letter]
```

Comment:
Let's go `address` here as well, since it's not to an individual person, or small group of people, but meant to sort of be spread abroad.